conf/trust.routes

```diff
@@ -1,10 +1,16 @@
 ->          /hmrc-frontend                                 hmrcfrontend.Routes
 GET         /assets/*file                                  controllers.Assets.versioned(path = "/public", file: Asset)
 
+
 GET         /:journeyId/sa-utr                             uk.gov.hmrc.minorentityidentificationfrontend.controllers.trustControllers.CaptureSautrController.show(journeyId)
 POST        /:journeyId/sa-utr                             uk.gov.hmrc.minorentityidentificationfrontend.controllers.trustControllers.CaptureSautrController.submit(journeyId)
 GET         /:journeyId/no-utr                             uk.gov.hmrc.minorentityidentificationfrontend.controllers.trustControllers.CaptureSautrController.noUtr(journeyId)
 
 GET         /:journeyId/self-assessment-postcode           uk.gov.hmrc.minorentityidentificationfrontend.controllers.trustControllers.CaptureSaPostcodeController.show(journeyId)
 POST        /:journeyId/self-assessment-postcode           uk.gov.hmrc.minorentityidentificationfrontend.controllers.trustControllers.CaptureSaPostcodeController.submit(journeyId)
 GET         /:journeyId/no-self-assessment-postcode        uk.gov.hmrc.minorentityidentificationfrontend.controllers.trustControllers.CaptureSaPostcodeController.noSaPostcode(journeyId)
+
+GET         /:journeyId/charity-hmrc-reference-number      uk.gov.hmrc.minorentityidentificationfrontend.controllers.trustControllers.CaptureCHRNumberController.show(journeyId)
```

Comment:
I don't think this is the correct route. Looks like on the ticket it's /CHRN

Reply:
Fixed